React/Views/RCTRefreshControlManager.m

```diff
@@ -15,6 +15,8 @@ @implementation RCTRefreshControlManager
 
 RCT_EXPORT_MODULE()
 
+#if !TARGET_OS_TV
```

Comment:
Prevent the whole interface and module from compiling instead. It's better to give a clear error message like "RCTRefreshControlManager doesn't exist" rather than having RCTRefreshControlManager vend RCTView's and nobody understanding why.